complaints/static/contact.html

```diff
@@ -55,5 +33,6 @@ <h4>Name : 91XXXXXXXXXX</h4>
 </div>
 </div>
 
+{% endblock %}
```

Comment:
closing body and html tags after endblock are in each file.So just put them in base.html to remove code duplicacy

Reply:
Also, Please squash all your commits in this branch into one before pushing.